### PR TITLE
Add transaction processing function

### PR DESCRIPTION
**Pull request by neithernut**
This PR adds a preliminary version of the function which processes transactions.
Targets #240.

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
WIP or ready for review?

**Comment by neithernut** [comment]
Ready for review.

File: src/action/manager.c [hunk 1]
```diff
@@ -25,5 +25,133 @@
...
+) {
+    struct ws_reply* retval = NULL;
+    int res;
+
+    // prepare the stack
+    struct ws_processor_stack stack;
+    res = ws_processor_stack_init(&stack);
+    if (res < 0) {
+        retval = (struct ws_reply*)
+                 ws_error_reply_new(transaction, -res, "Could init stack",
```
Comment (matthiasbeyer) [inline src/action/manager.c]: "Could" or "Couldn't"?

**Comment by matthiasbeyer** [comment]
ACK.

**Pull request merged**